R/backend-dbplyr__duckdb_connection.R

```diff
@@ -302,6 +302,17 @@ sql_translation.duckdb_connection <- function(con) {
         build_sql("DATEDIFF('day', ", !!start, ", " ,!!end, ")")
 
       },
+      date_build = function(year, month = 1L, day = 1L, ..., invalid = NULL) {
+        dbplyr:::check_unsupported_arg(invalid, allow_null = TRUE)
+        rlang::check_dots_empty()
+        build_sql("MAKE_DATE(CAST(", year, " AS INTEGER), CAST(", month, " AS INTEGER), CAST(", day, " AS INTEGER))")
```

Comment:
I see how `SELECT MAKE_DATE(2022.0, 3.0, 8.0);` fails in duckdb. Still, casting shouldn't be a concern to `MAKE_DATE()` .